index.js

```diff
@@ -86,7 +86,8 @@ function wrapInControlFlow(globalFn, fnName) {
               // ignored.
               var proxyDone = fulfill;
               proxyDone.fail = function(err) {
-                reject(err);
+                var wrappedErr = new Error(err);
```

Comment:
When a test does `done.fail('message in a string')`, we need this to capture the stack trace. Otherwise, it's just the trace from the `done.fail` call below on line 101.